src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -35,6 +38,11 @@ public MarkdownV2Renderer(ParserMode mode, int maxSyntaxWidth)
             this._mode = mode;
         }
 
+        public MarkdownV2Renderer(int maxLineWidth)
```

Comment:
Add a default ctor overload that will call this ctor.
User should be able to instantiate rendered with default (recommended) width value.